homeassistant/components/apple_tv.py

```diff
@@ -91,7 +91,7 @@ def configuration_callback(callback_data):
         hass.async_add_job(configurator.request_done, instance)
 
     instance = configurator.request_config(
-        hass, 'Apple TV Authentication', configuration_callback,
+        'Apple TV Authentication', configuration_callback,
```

Comment:
I made a change since you implemented it https://github.com/armills/home-assistant/blame/4b6412874ebe3a7556fe3369194be0b344e74e13/homeassistant/components/apple_tv.py#L68

The idea is that when you refer to any component under `hass.components`, any method that is annotated with `@bind_hass` will automatically get `hass` filled in as first argument.

By keeping the hass argument there, we would pass in `hass` twice which cannot work.